cluster/hooks.go

```diff
@@ -32,7 +32,9 @@ import (
 	"github.com/banzaicloud/pipeline/internal/security"
 	pkgCluster "github.com/banzaicloud/pipeline/pkg/cluster"
 	pkgCommon "github.com/banzaicloud/pipeline/pkg/common"
+	helm2 "github.com/banzaicloud/pipeline/pkg/helm"
 	pkgHelm "github.com/banzaicloud/pipeline/pkg/helm"
```

Comment:
maybe we should rename this to helm2 because in every other place this import called that way

Reply:
thx, fixed